memstore/common/vector_party.go

```diff
@@ -201,17 +201,8 @@ type ArchiveVectorParty interface {
 	SliceIndex(lowerBoundRow, upperBoundRow int) (startIndex, endIndex int)
 }
 
-// ListDataValueReader defines how to read ith element in jth row from a data source.
-// Out of boundary access will panic.
-type ListDataValueReader interface {
-	ReadElementValue(row int, i int) unsafe.Pointer
-	ReadElementBool(row int, i int) bool
-	ReadElementValidity(row int, i int) bool
```

Comment:
we don't have element validity any more?

Reply:
element is not needed in go lang, there is a ArrayValueReader which is only for testing purpose